package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.3",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.26.1",
```

Comment:
I think we could continue using fetch for this project. I used fetch for my tasks. Though there is no issue of using axios but I think it would be good if we could communicate about it because its impact is beyond the task requirement.

Reply:
Okay, we could communicate about the next time we meet